021-quarkus-panache-multiple-pus/README.md

```diff
@@ -0,0 +1,8 @@
+# Quarkus - multiple persistence units
+This scenario uses a pair of persistence units, based upon MariaDB and PostgreSQL datasource.
+
+To run the required databases locally, run the following containers:
+```
+docker run -p 127.0.0.1:3306:3306 --name mariadb -e MYSQL_USER=mardiadb -e MYSQL_PASSWORD=mariadb -e MYSQL_DATABASE=mariadb -d mariadb:10.3.6
```

Comment:
It won't connect in dev mode
Please check what can be done to run it locally.
MYSQL_ROOT_PASSWORD should be specified.

Reply:
The containers are a bit of pain to set up locally, and I don't think this should block the test suite for now. What do you think?